doc/setup_instructions.md

```diff
@@ -486,3 +485,117 @@ build tool option, if it isn't already.
 
 Your locally cloned Thrive version may get messed up from time to time.
 [Here are the steps to fix it.](https://wiki.revolutionarygamesstudio.com/wiki/Cleaning_Local_Thrive_Version)
+
+Working with translation
```

Comment:
I think this should be put in a new file called `working_with_translations.md`, with just a relative link contained in the additional tips here.
Also could you then add a link to that new file from the main readme and contributing info markdown files?